src/containers/MapContainer/PointClusterLayer.tsx

```diff
@@ -57,14 +57,14 @@ const PointClusterLayer: React.FC<PointClusterLayerProps> = ({ onItemSelected })
       const isMarker = !f.getBounds;
       if (!f.feature) return;
       const featureId = String(isMarker ? f.feature.id : f.feature.properties.id);
-      const zoom = isMarker ? 16 : 14;
-      if (featureId === id && source === f.feature.contact) {
+      if (featureId === id) {
```

Comment:
This won't work because there are duplicated id's in the layer, het match should be done on contact as well. The contact is set in `services/layer-aggregator/layersConfig.js`. 
As an improvement I would set the feature.properties.id to feature.id in the layersConfig.js transformers for uniformity

Reply:
it is back in there